src/applications/caregivers/config/chapters/signAsRepresentative/uploadPOADocument.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+// import fullSchema from 'vets-json-schema/dist/10-10CG-schema.json';
+import { representativeFields } from 'applications/caregivers/definitions/constants';
+import fileUploadUI from 'platform/forms-system/src/js/definitions/file';
+import environment from 'platform/utilities/environment';
+
+// const { representative } = fullSchema.properties;
+// const veteranProps = veteran.properties;
+// const { address, phone } = fullSchema.definitions;
+
+const DocumentUploadDescription = () => {
+  return (
+    <div>
+      <p>
+        If you’re signing as a legal representative, you can upload supporting
+        documents showing your authority to complete this application on behalf
+        of the Veteran.
+      </p>
+
+      <legend style={{ padding: 0 }}>
+        Upload your supporting documentation
+      </legend>
+      <p>
+        You can upload document in a .pdf, .jpeg, or .png file format. You’ll
+        first need to scan a copy of your document onto your computer or mobile
+        phone. You can upload the document from there.
+      </p>
+
+      <p>Guidelines for uploading a file:</p>
+      <ul>
+        <li>File types you. can upload: .pdf, .jpeg, or .png</li>
+        <li>Maximum file size: 25MB</li>
+      </ul>
+
+      <i>
+        A 1MB file equals about 500 pages of text. A photo is usually about 6MB.
+        Large files can take longer to upload with a slow internet connection.
+      </i>
```

Comment:
Semantically, it's better to use `<em>` instead of `<i>`

Reply:
Hmmm so I am all for either or I was thinking about it as I was writing this code and I found that the `<i>` is just italics and the `<em>`  was more of an emphasis to signify importance... then I got caught up on how important is this information? and I was not really sure. All for making it more emphasized.